education/blockchain/mining.md

```diff
@@ -1,45 +1,2 @@
-
-The next morning, you finally get the next number first! Your ingenuity paid off! But every kid in the class has the same calculator, and they all start writing their own programs. The teacher sees this and says, "Aha! I see that you've started automating the search! Well the rules are changing a little!"
-
-You watch as the teacher erases the simple equation on the board and writes down a huge long equation. "Find x where x to the power of x divided by 57 plus the square root ..." It goes on and on!
-
-"Your little calculators might not be enough for this!"
-
-When you get home you program the formula on your desktop computer. But it's still just a matter of trying a number, running it through the formula to see if you end up with a result that matches the teacher's requirement. This time the goal is get as close to zero as possible. "0.09 or less, but not negative".
-
-Your computer starts running the calculations. It checks a number, then adds 1 and checks that number. Then adds 1 and checks that number. It runs for an hour, and nothing works! You give up and go to bed.
-
-The next morning, one of the other students named Sarah proudly claims the prize! "How did you do it? I ran the calculations for an hour and nothing worked!"
-
-Sarah says "I just left it running all night! There was an answer when I woke up."
-
-## Proof of Work Vs. Proof of Stake
-Imagine you are at the theme park and there are two rollercoasters. Getting to ride the coaster is your reward. 
-
-If you want to ride the PoW rollercoaster, then you have to compete in a race every 10 minutes. Whoever can run fastest wins the race and gets to ride. The same people tend to be fastest every race, and always get to ride. Slower runners rarely get a ride. People started forming relay teams (pools) to get a chance on the coaster, to split the reward.
-
-The PoS rollercoaster is different, there are a bunch of queues in parallel, if you are rich you go in the rich queue and get to the front faster, but no matter how poor you are you always get a ride in the end. Who is chosen from the front of those parallel queues is random, but the VIP queue has less people in it so they progress to the front quicker. The PoS rollercoaster uses (how much money you have x time since last ride) to build 'stake weight'. The higher your stake weight, the faster you get to the front. Once you ride the coaster, your stake weight is set to zero and you go back to the back of the queue system.
+Mining is how new transactions get added to a blockchain. There are two main types of mining, Proof of Work (PoW) and Proof of Stake (PoS)
```

Comment:
Why are we removing the mining analogy? It's a very complex subject and is one that definitely requires one.